app/src/main/res/layout/fragment_splash.xml

```diff
@@ -1,60 +1,67 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.core.widget.NestedScrollView xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
I don't think we had a problem on this screen but just to make sure it works anyway. 

I checked if we increase the text size insanely large and the screen size is ridiculously small then we have an overlap between the logo and the text so let's keep it just for extreme edge cases :)

Reply:
tested on device (Pixel 2 XL) with system font set to Largest, system display size set to Largest, and app put into split screen. in this case, without the NestedScrollView the "Getting Started" button if off screen and unclickable